packages/governance-pool/contracts/GovernancePool.sol

```diff
@@ -34,9 +34,15 @@ contract GovernancePool is ERC1155Holder, Context, IGovernancePool {
     address internal immutable INVESTMENT_POOL_FACTORY_ADDRESS;
     uint8 internal immutable VOTES_PERCENTAGE_THRESHOLD;
     uint256 internal immutable VOTES_WITHDRAW_FEE; // number out of 100%; E.g. 1 or 13
-
-    /// @notice mapping from investment pool id => status
-    mapping(uint256 => InvestmentPoolStatus) internal investmentPoolStatus;
+    // TODO: investment pool also hold the values of state. Only in one contract value should be hardcode and others should get from it.
+    // This will prevent for later bugs when changing state values
+    uint256 internal constant FUNDRAISER_ONGOING_STATE_VALUE = 4;
```

Comment:
Decide where to actually put it.
What contract will be the source of truth?
And maybe there is a better solution.